gateway/1.4/modules/ROOT/pages/_partials/task-reg-run-flex-gateway.adoc

```diff
@@ -175,9 +178,9 @@ If `split` is set to `true`, registration information is split into two files: `
 // end::replace-content[]
 // tag::reg-command-heading[]
 
-=== Registration Command
+=== Register Flex Gateway
 
```

Comment:
it wasn't clear why they would replace the info provided in this section since it seems intended for the command in the next section, but the options listed here aren't in the format you'd use in the command, so the reason for replacing the sample content wan't really clear in this section, especially if you haven't looked at the next section yet